src/test/java/AddRemoveTest.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import java.util.List;
+
+public class AddRemoveTest extends BaseTest {
+    @BeforeMethod
```

Comment:
It's better to add @before/after annotations to setuo() and exit() methods in BaseTest class so you won't need to call them each time in each nested  class via super.setUp() but they will be executed automatically